android/app/build.gradle

```diff
@@ -247,7 +247,7 @@ android {
         release {
             // Caution! In production, you need to generate your own keystore file.
             // see https://reactnative.dev/docs/signed-apk-android.
-            signingConfig signingConfigs.debug
+            // signingConfig signingConfigs.debug
```

Comment:
what is the reason for commenting this out? would it make sense to remove it if we do not need it?

Reply:
The singer started to complain about it. We don't need a debug signature in a release packages. Removed.